sky/specs/style-guide.md

```diff
@@ -8,44 +7,114 @@ that everyone, whether reading the code for the first time or
 maintaining it for years, can quickly determine what the code does. A
 secondary goal is avoiding arguments when there are disagreements.
 
-All languages
--------------
+## All languages
 
-Avoid checking in commented-out code. It will bitrot too fast to be
-useful, and will confuse people maintaining the code.
+### AVOID checking in commented-out code
```

Comment:
I think the Dart style guide using capital letters in the headings is pretty ugly, but the way the Dart style guide is organised they first have headings for each topic, and then have headings that barely look like more than emphasised paragraphs.

I think we could find a middle ground where we have topic headings, then subheadings for each major guideline, but where the guidelines are written as full sentences with normal capitalisation (rather than shoehorning them into this "DO that" "PREFER this" style).